README.md

```diff
@@ -1,2 +1,18 @@
 # switchboard-samples
 Sample senders and receivers to be used with the Switchboard project. https://github.com/felixlapierre/switchboard
+
+# Requirements
+* [Python](https://www.python.org) 3.7.3+
+* [ffmpeg](https://ffmpeg.org) 4.1.6+
+
+# Usage
+1. Create your virtual environment and install the required packages  
+`python3 -m venv env`  
+`source env/bin/activate`  
+`pip install -r requirements.txt`  
```

Comment:
Some clarification for those unfamiliar with python virtual environments:

- The first step creates a virtual environment called env (you can name it however you want, that's just how I named it)
- The second step activates the virtual environment you just created (you'll see your shell's prompt change)
- The last step installs all the packages from the requirements.txt file into your virtual environment (after that step you can run `pip list` to see which packages you currently have installed)

Now I realized that activating the virtual environment (step 2) is done differently on Windows and Linux so that's my bad, I'll change the README accordingly (basically for Windows it's `env\Scripts\activate.bat`)